app/views/decks/_deck_summary.html.erb

```diff
@@ -0,0 +1,106 @@
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-published"><%= resource.where(status: 'published').count %> Published</span></li>
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-unpublished"><%= resource.where(status: 'unpublished').count %> Unpublished</span></li>
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-paid"><%= resource.where(tier: 'paid').count %> Paid</span></li>
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-free"><%= resource.where(tier: 'free').count %> Free</span></li>
+                    </a>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-bookmark" aria-hidden="true"></span>Categories</th>
+                <td><%= @categories.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <% @categories.each do |category| %>
+                      <li class="list-item-label">
+                        <span class="label label-default"><%= @category_deck_links.where(category_id: category.id).count %> <%= category.name %></span>
+                      </li>
+                    <% end %>
+                  </ul>
+                </td>
+              </tr>
+
+            <% elsif resource == @cards %>
+              
+              <tr>
+                <th><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>Created At</th>
+                <td>3/20/2016</td>
+                <td></td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-book" aria-hidden="true"></span>Cards</th>
+                <td><%= resource.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <a href="#"><li class="list-item-label"><span class="label label-needs-review">0 Needs Review</span></li></a>
+                    <a href="#"><li class="list-item-label"><span class="label label-published">0 Published</span></li></a>
+                    <a href="#"><li class="list-item-label"><span class="label label-unpublished">0 Unpublished</span></li></a>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-bookmark" aria-hidden="true"></span>Categories</th>
+                <td>2</td>
+                <td>
+                  <ul class="list-inline">
+                    <li class="list-item-label"><span class="label label-default">25 Category One</span></li>
+                    <li class="list-item-label"><span class="label label-default">25 Category Two</span></li>
+                  </ul>
+                </td>
```

Comment:
What exactly are we trying to express in this `<td>` block? Cards are not associated with `categories` as you will notice within the `category.rb` model
```ruby
class Category < ActiveRecord::Base
  has_many :category_deck_links
  has_many :decks, through: :category_deck_links
```
May just be a matter of removing the numbers and listing all the categories for a deck